inst/pages/04_containers.qmd

```diff
@@ -675,7 +675,7 @@ Let us add `colData` from the sample metadata, which is stored in a CSV file.
 
 ```{r}
 # CSV file with colnames in the first row and rownames in the first column
-sample_meta <- read.csv(sample_meta_file_path,
+col.data <- read.csv(sample_meta_file_path,
```

Comment:
As this is variable, do not change it. (Moreover, I think variable_name is better than variable.name since we should have this harmonized across the book)